geopandas/explore.py

```diff
@@ -276,6 +277,22 @@ def my_colormap(value):  # scalar value defined in 'column'
             "'conda install -c conda-forge folium matplotlib mapclassify' "
             "or 'pip install folium matplotlib mapclassify'."
         )
+    # prepare for change in matplotlib
+    try:
+        from matplotlib import colormaps as cm_
+
+        cm_["viridis"].resampled(10)
+
+        def _cm(cmap, lut=None):
+            cmap = "viridis" if not cmap else cmap
+            cm = cm_[cmap]
+            return cm.resampled(lut) if lut else cm
+
+    except (ImportError, AttributeError):
+        import matplotlib.cm as cm_
+
+        def _cm(cmap, lut=None):
+            return cm_.get_cmap(cmap, lut)
```

Comment:
Can we have this PR focusing on array-like parameters? And leave this for another one.

Reply:
stripped out